src/game_api/entity.cpp

```diff
@@ -413,8 +405,8 @@ std::tuple<float, float, uint8_t> get_position(uint32_t uid)
 std::tuple<float, float, uint8_t> get_render_position(uint32_t uid)
 {
     Entity* ent = get_entity_ptr(uid);
-    if (ent)
-        return std::make_tuple(ent->position_render().first, ent->position_render().second, ent->layer);
+    if (ent && !ent->rendering_info->stop_render)

```

Comment:
Why? Assuming `stop_render` means that the entity shall not be rendered. That does seem to imply that we should not return the given position here. Is there a good motivation for this?

Reply:
there is negation there "!" so it will return position if the `stop_render` is false

Reply:
Render stuff is not exposed, so there is no way telling if the returned value from this function is valid or not